common/supplemental/windowsZones.xml

```diff
@@ -246,13 +246,19 @@ For terms of use, see http://www.unicode.org/copyright.html
 			<!-- (UTC-03:00) Salvador -->
 			<mapZone other="Bahia Standard Time" territory="001" type="America/Bahia"/>
 			<mapZone other="Bahia Standard Time" territory="BR" type="America/Bahia"/>
-
-			<!-- (UTC-02:00) Coordinated Universal Time-02 -->
-			<mapZone other="UTC-02" territory="001" type="Etc/GMT+2"/>
-			<mapZone other="UTC-02" territory="BR" type="America/Noronha"/>
-			<mapZone other="UTC-02" territory="GS" type="Atlantic/South_Georgia"/>
-			<mapZone other="UTC-02" territory="ZZ" type="Etc/GMT+2"/>
-
+      
+      <!-- (UTC-02:00) Mid-Atlantic Standard Time -->
+      <mapZone other="UTC-02" territory="001" type="Etc/GMT+2"/>
+      <mapZone other="UTC-02" territory="BR" type="America/Noronha"/>
+      <mapZone other="UTC-02" territory="GS" type="Atlantic/South_Georgia"/>
+      <mapZone other="UTC-02" territory="ZZ" type="Etc/GMT+2"/>
+      
+      <!-- (UTC-02:00) Mid-Atlantic Standard Time -->
```

Comment:
The `"Mid-Atlantic Standard Time"` zone has been deprecated in Windows, and thus should not appear in the CLDR.  It is marked with `IsObsolete` in the Windows registry.

Any system having reference to `"Mid-Atlantic Standard Time"` should be updated to use `"UTC-02"` instead.